fem/bilinearform.hpp

```diff
@@ -224,7 +222,7 @@ class BilinearForm : public Matrix
 
    /** @brief Check if static condensation was actually enabled by a previous
        call to EnableStaticCondensation(). */
-   bool StaticCondensationIsEnabled() const { return static_cond; }
+   bool StaticCondensationIsEnabled() const { return static_cond.get() != nullptr; }
```

Comment:
(minor) to help my understanding of best practices, are you not doing just `return static_cond` for readability? Or is there a case where `static_cond` is `nullptr` by design? Something else?

Reply:
We cannot just do `return static_cond` because the `unique_ptr` [boolean conversion operator](https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_bool) is `explicit`.

However, `unique_ptr` does have a comparison to `nullptr`, which simplifies this a bit. Changed in 4979dc1701e2b9d1ac3159ff55bdf4155fb5654e.